portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -38,7 +38,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PreparedQuery results = datastore.prepare(query);
     List<String> shows = new ArrayList<>();
     for (Entity entity : results.asIterable()) {
-      shows.add((String)entity.getProperty("show"));
+      shows.add(entity.getProperty("show").toString());
```

Comment:
No need to even add the toString, that's already implicitly called when it realizes that it's trying to be set as a String.
So just `shows.add(entity.getProperty("show"));`

Reply:
Get an error doing that.  'java.lang.Object cannot be converted to java.lang.String'

Reply:
That's interesting... Definitely not the expected behavior. Alright we can leave the toString in there.
Normally, Objects and its inheritors have their toString behave as shown in this [article](https://www.baeldung.com/java-tostring). Not too sure what the exception is here.